submissions/medinatapanpa/php/php/footer.php

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Since this is a partial element on the page it shouldn't have a complete html declaration.
Can you also try to add content? It should not be added if it is empty